README.md

```diff
@@ -93,6 +97,20 @@ The basic operation of the tool is this:
 The data can then be loaded by the ilib-localedata package for use in the various ilib-*
 packages.
 
+## Basic Operation for legacy version of ilib
+1. Write the `inc` file to specify which files to assemble.
+   If not, the default file that the tool (`./src/ilib-ilib-assemble-inc.js`)has will be used.
```

Comment:
"If not, the default file `./src/ilib-ilib-assemble-inc.js` will be used which assembles all ilib classes and data together. If you only need a subset of ilib, copy this file and delete the parts you do not need.

Reply:
I updated it